React/Modules/RCTAlertManager.m

```diff
@@ -16,22 +16,28 @@
 
 @implementation RCTConvert (UIAlertViewStyle)
 
-RCT_ENUM_CONVERTER(UIAlertViewStyle, (@{
-  @"default": @(UIAlertViewStyleDefault),
-  @"secure-text": @(UIAlertViewStyleSecureTextInput),
-  @"plain-text": @(UIAlertViewStylePlainTextInput),
-  @"login-password": @(UIAlertViewStyleLoginAndPasswordInput),
-}), UIAlertViewStyleDefault, integerValue)
+RCT_ENUM_CONVERTER(RCTAlertViewStyle, (@{
+  @"default": @(RCTAlertViewStyleDefault),
+  @"secure-text": @(RCTAlertViewStyleSecureTextInput),
+  @"plain-text": @(RCTAlertViewStylePlainTextInput),
+  @"login-password": @(RCTAlertViewStyleLoginAndPasswordInput),
+}), RCTAlertViewStyleDefault, integerValue)
 
 @end
 
+#if TARGET_OS_TV
+@interface RCTAlertManager()
+#else
 @interface RCTAlertManager() <UIAlertViewDelegate>
```

Comment:
I think I just forgot to kill all of the UIAlertView related code in https://github.com/facebook/react-native/commit/07553d0f1c61173edad804c4e7a93c902719e05b

I'll either get to it later or feel free to kill it here too.